ga4gh/cli.py

```diff
@@ -428,16 +429,50 @@ def run(self):
 
     def _textOutput(self, gaObjects):
         """
-        Prints out the specified Variant objects in a VCF-like form.
+        Prints out the specified Read objects.
         """
         for read in gaObjects:
             # TODO add in some more useful output here.
             print(read.id)
 
 
+class SearchGenotypePhenotypeRunner(AbstractSearchRunner):
+    """
+    Runner class for the genotypephenotype/search method.
+    """
+    def __init__(self, args):
+        super(SearchGenotypePhenotypeRunner, self).__init__(args)
+        argStore = vars(args)
+        for key, value in vars(args).iteritems():
```

Comment:
I'm slightly confused by this --- is the idea to parse the argument for these values into a JSON dict? I'd try to be a bit more explicit about it
```python
def parseJson(value):
     ret = None
     if value is not None:
          ret = json.loads(value)
     return ret
self._feature = parseJson(args.feature)
# etc
```